pytest_fixtures/core/contenthosts.py

```diff
@@ -143,3 +144,17 @@ def container_contenthost(rhel7_contenthost, default_sat):
         rhel7_contenthost.execute(f'yum -y install {service}')
         rhel7_contenthost.execute(f'systemctl start {service}')
     return rhel7_contenthost
+
+
+@pytest.fixture(scope='module')
+def set_remote_execution_by_default(request):
+    """Fixture to set the 'remote_execution_by_default' setting"""
+    if getattr(request, 'param', None) is not None:
+        value = request.param  # indirect parametrization value
+    else:
+        value = True  # default value
+    remote_execution_setting = entities.Setting().search(
```

Comment:
If you switch this fixture to use the a target_sat fixture object, then it can be used in destructive tests as well.
```suggestion
    remote_execution_setting = module_target_sat.api.Setting().search(
```